send_errors option of SoapServer added

Kerion opened the pull request:
send_errors option of \SoapServer added (http://php.net/manual/de/soapserver.soapserver.php)

Without this option, Fatal errors are always printed out by \SoapServer

Kerion commented:
Whitespace-Check is failing like in PullRequest #5 

I don't know, how to fix it, because I think the Whitespaces are correct.

weierophinney commented:
@Kerion Run `./vendor/php-cs-fixer fix --config-file=.php_cs` from the repository root, and it will fix the issues. You'll need to add, commit, and push the changes when done.

Kerion commented:
@weierophinney thanks for the info but it did not help:

`
root@**:/**/zend-soap# git clone https://github.com/Kerion/zend-soap.git
[]
root@**:/**/zend-soap# php composer.phar update
[]
root@**:/**/zend-soap# ./vendor/bin/php-cs-fixer fix --config-file=.php_cs
Loaded config from ".php_cs"
Fixed all files in 19.869 seconds, 10.500 MB memory used
root@**:/**/zend-soap# git commit -m "php-cs-fixer"
# On branch master
nothing to commit (working directory clean)

root@**:/**/zend-soap# ./vendor/bin/php-cs-fixer fix -v --diff --dry-run
Loaded config from "/home/dk/zend-soap/zend-soap/.php_cs"
....................................
Legend: ?-unknown, I-invalid file syntax, file ignored, .-no changes, F-fixed, E-error
Fixed all files in 18.072 seconds, 10.250 MB memory used

root@**:/**/zend-soap# git commit -m "php-cs-fixer"
# On branch master
nothing to commit (working directory clean)
`


The lines where phpcs is failing in Travis i didn't even touch in my commit....

Kerion commented:
ok, got it, wrong branch :(

Kerion commented:
build succeded in #20
